platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/components/PcesWriterWiring.java

```diff
@@ -33,7 +33,7 @@
  *                                                 complete
  * @param eventInputWire                           the input wire for events to be written
  * @param discontinuityInputWire                   the input wire for PCES discontinuities
- * @param nonAncientEventWindowInput               the input wire for non ancient event windows
+ * @param eventWindowInput               the input wire for non ancient event windows
```

Comment:
doc mentions "non ancient event windows" (and is misaligned)

Reply:
fixed